.github/ISSUE_TEMPLATE/other.yaml

```diff
@@ -2,6 +2,192 @@ name: Other issue
 description: Create a new issue to help us improve the collector
 labels: ["needs triage"]
 body:
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component(s)
+      description: Which component(s) does your issue concern?
+      multiple: true
+      options:
+      # Start Collector components list
```

Comment:
Can we insert a note here that the list is autogenerated and should not be manually changed?

Reply:
Good call. Done.